sources/Inputs/src/NetworkManager.cpp

```diff
@@ -68,6 +71,11 @@ void NetworkManager::buildTree() {
     const auto &shunts = networkVL->getShuntCompensators();
     std::unordered_map<Node::NodeId, std::vector<Shunt>> shuntsMap;
     for (const auto &shunt : shunts) {
+      if (shunt->hasInitialConditions()) {
```

Comment:
I have the feeling that we could factorize this into a method taking a boolean as parameter to avoid code duplication